mbuild/compound.py

```diff
@@ -607,17 +610,16 @@ def save_gsd(self, filename, structure, forcefield_name,
         write_gsd(structure, filename, forcefield, box, **kwargs)
 
     def save_gromacs(self, filename, structure, forcefield_name,
-                      forcefield_files, box, **kwargs):
+                     forcefield_files, box, **kwargs):
         """ """
         # Create separate file paths for .gro and .top
         filepath, filename = os.path.split(filename)
         basename = os.path.splitext(filename)[0]
         top_filename = os.path.join(filepath, basename + '.top')
         gro_filename = os.path.join(filepath, basename + '.gro')
+        #  TODO: I think  the forcefield varable can be deleted here
```

Comment:
Yeah. The whole group of savers needs some cleanup related to boxes and forcefields